braindecode/datasets/datasets.py

```diff
@@ -44,6 +47,22 @@ def _fetch_and_unpack_moabb_data(dataset, subject_ids):
     return raws, description
 
 
+def _annotations_from_raw(raw, dataset):
```

Comment:
I would rename this function to something like `_annotations_from_stim_channel` as it specifically requires stim channels to be available.

Reply:
Yes, I agree.